src/lambda.d.ts

```diff
@@ -78,7 +78,7 @@ export interface AlphaClientConfig {
 }
 
 export interface AlphaClient extends AxiosInstance {
-  new (config: AlphaClientConfig);
+  new (config: AlphaClientConfig): AxiosInstance;
```

Comment:
I believe that the default return type for `new` on a interface is the interface itself. In this case it would be `AlphaClient` which seems right. 

Your new typing says that `new AlphaClient(options)` would return a `AxiosInstance` type, which seems wrong. Can you share some code that wasn't work for you?

Reply:
Some examples are at https://www.typescriptlang.org/docs/handbook/interfaces.html.

If you are use you want to keep a return type, then I'd suggest returning `AlphaClient`